pinot-controller/src/main/java/org/apache/pinot/controller/util/ServerInfoFetcher.java

```diff
@@ -53,7 +53,7 @@ private ServerInfo getServerInfoOndemand(String instanceId) {
     List<String> tags = instanceConfig.getTags();
     ZNRecord record = instanceConfig.getRecord();
     boolean helixEnabled = record.getBooleanField(
-        InstanceConfig.InstanceConfigProperty.HELIX_ENABLED.name(), false);
+        InstanceConfig.InstanceConfigProperty.HELIX_ENABLED.name(), true);
```

Comment:
@sajjad-moradi @jasperjiaguo The default value of `HELIX_ENABLED` must be true. Otherwise `ControllerPeriodicTasksIntegrationTest` will fail